### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/chart/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#838] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/chart/

该插件提供了 6 种方法让您将图表插入到 Markdown 文件中。 Chart.js：一个轻量级、易于使用、高度可定制的图表库。 与 ECharts 相比，Chart.js 更轻量。 ECharts：一个强大的、交互式的浏览器图表和可视化库。 与 Chart.js 相比，ECharts 更强大。 flowchart: Markdown 扩展，用于生成...

https://theme-hope.vuejs.press/zh/guide/markdown/chart/

**Comment by matiding2016** [comment]
Markmap的生成高度是否可以控制？目前如果内容较多的话，默认的展示就会比较小
